packages/plugin-ethereum/src/EthereumChildPlugin.ts

```diff
@@ -47,10 +49,19 @@ export class EthereumChildPlugin extends ChildPlugin<EthereumChildPluginConfig>
           this.logger("middleware error: ", err);
         });
     } else if (action.type === "PARENT_REQUEST" && action.data.type === "WEB3_ENABLE") {
-      this.dispatch({
-        type: REQUEST_VIEW_ACTION,
-        payload: { route: "/ethereum/web3enable", requestID: action.requestID },
-      });
+      if (this.config.burnerPreference === "always") {
+        const burnerProvider = new BurnerProvider({
+          rpcUrl: this.config.rpcURL, // oof I don't like the difference in caps here rpcUrl is the standard
+          namespace: this.dependencies.iframe.parentDomain
+        });
+        this.activateWeb3(burnerProvider, "Burner Wallet", action.requestID);
+      } else {
+        console.log("ENABLE!!!!",this.config)
```

Comment:
either remove the console.log or change to use this.logger (so it can be turned off in production)